components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -193,19 +193,15 @@
 
     _initUI: function() {
       var generateSection = function(id, title, intro) {
-         return new Element('div', {'id': id, 'class' : 'section'})
-                    .insert(title &amp;&amp; new Element('h2').update(title) || '')
-                    .insert(intro &amp;&amp; new Element('p', {'class' : 'intro xHint'}).update(intro) || '')
-                    .insert(new Element ('div', {'class' : 'section-contents'}));
+         var section = new Element('div', {'id': id, 'class' : 'section'});
+         if (title) {section.insert(title &amp;&amp; new Element('h2').update(title) || '');}
+         if (intro) {section.insert(intro &amp;&amp; new Element('p', {'class' : 'intro xHint'}).update(intro) || '');} 
+         section.insert(new Element ('div', {'class' : 'section-contents'}));
+         return section;
       };
 
-      var serverTitle = this._pushManyPatients ? "$services.localization.render('phenotips.patientSharing.push.dialog.serverSelection.label.multiplePatients')":
-                                                "$services.localization.render('phenotips.patientSharing.push.dialog.serverSelection.label.singlePatient')";
-      var serverDesc  = this._pushManyPatients ? "$services.localization.render('phenotips.patientSharing.push.dialog.serverSelection.hint.multiplePatients')":
-                                                 "$services.localization.render('phenotips.patientSharing.push.dialog.serverSelection.hint.singlePatient')";
       this._container = new Element('form', {'id' : 'push-patients-ui', 'class' : 'xform', 'method' : 'post', 'action' : this._serviceURL});
-      this._container.insert(generateSection('server-selection', serverTitle, serverDesc))
-                     .insert(generateSection('user-selection', "$services.localization.render('phenotips.patientSharing.push.dialog.authentication.label')","$services.localization.render('phenotips.patientSharing.push.dialog.authentication.hint')"))
+      this._container.insert(generateSection('user-selection', "$services.localization.render('phenotips.patientSharing.push.dialog.authentication.label')","$services.localization.render('phenotips.patientSharing.push.dialog.authentication.hint')"))
```

Comment:
Would be nice to start using `$!escapetool.javascript(...)` to escape all translated text when we modify js code, even if it wasn't done originally.